libkbfs/block_retrieval_queue.go

```diff
@@ -173,7 +173,7 @@ func (brq *blockRetrievalQueue) Request(ctx context.Context, priority int, kmd K
 			// specific type where the request blocks are CommonBlocks, but
 			// that direction can Set correctly. The cache will never have
 			// CommonBlocks.
-			cachedBlock, hasPrefetched, err :=
```

Comment:
You should probably do the same thing here. Although we'll have a race for Permanent -> Transient changes.

When we `DeletePermanent`, should we automatically put the block into `Transient` under lock?